src/videojs-vjstranscribe.js

```diff
@@ -91,25 +91,32 @@ class VjsTranscribe extends Plugin {
 
   }
 
+  getTextTracks() {
+    let tracks = this.player.textTracks().tracks_;
+
+    return tracks.filter((track) => {
+      return track.kind !== 'metadata';
+    })
+  }
+
   getActiveTrack() {
     let i, track;
-    for (i = 0; i < this.player.textTracks().tracks_.length; i++) {
-      track = this.player.textTracks().tracks_[i];
+    for (i = 0; i < this.getTextTracks.length; i++) {
```

Comment:
This should be `this.getTextTracks().length`

Reply:
ah yeah nice, thanks